csrc/device_lower/analysis/predicate_elimination.cpp

```diff
@@ -1035,13 +1036,27 @@ Val* PredicateElimination::getInitValue(TensorView* tv) const {
 
 std::string PredicateElimination::toString() const {
   std::stringstream ss;
-  ss << "Tensors that do not need predication:";
+  VectorOfUniqueEntries<TensorView*> non_predicated_tvs;
```

Comment:
I guess you didn't use `set<TensorView*>` because you like to print them in the order that they appear in the fusion?

Reply:
Yes